CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Grafana Mimir - main / unreleased
+* [ENHANCEMENT] Upgrade Docker base images to `alpine:3.16.0`. #2028
```

Comment:
Sorry, I think I didn't myself clear enough. Previously, the CHANGELOG entry was around line 700, in the CHANGELOG section of a previous version. You should move this to entry to line 29, at the end of the current `[ENHANCEMENT]` block, related to the "main / unreleased" changes.

Reply:
my fault, I see it now, serves me right for making changes prior to having coffee. Thanks for the help reviewing!